meson.build

```diff
@@ -6,7 +6,8 @@ project('SU2', 'c', 'cpp',
         default_options: ['buildtype=release',
                           'warning_level=0',
                           'c_std=c99',
-                          'cpp_std=c++17'])
+                          'cpp_std=c++11',
```

Comment:
Why the change of c++ from 17 to 11? Is it required for Tracy?  From the documentation I think that c++11 is the lowest version possible. I think c++17 should remain the default and, if necessary, the c++11 will be set when running meson if tracy is on.